src/ensembl/src/content/app/entity-viewer/interstitial/EntityViewerInterstitial.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import EntityViewerInterstitialInstructions from './entity-viewer-interstitial-instructions/EntityViewerInterstitialInstructions';
+import AppBar from 'src/shared/components/app-bar/AppBar';
+
+const EntityViewerInterstitial = () => {
+  return (
+    <div>
+      <AppBar
+        appName={'Entity Viewer'}
+        mainContent={'To start using this app...'}
```

Comment:
Like I mentioned above we might need some consensus on what to use for attribute values that are static strings